amundsenatlastypes/schema/06_column_partition_schema.json

```diff
@@ -2,28 +2,28 @@
   "entityDefs": [
```

Comment:
@dwarszawski @morpheyesh this is what I was talking about. 
Do we need to have a relationship between hive_column and hive_column_partition ? I don't think so. The only relation needed would be hive_colum_partition with hive_table_partition. 
So we can simply add "hive_column" as the superType, and it will all work just fine. No relation needed. 
"hive_column" is already inherited from "Column" which has the statistics and other required changes in place. 


P.S: And also, this is only hive_column_partition, so maybe change the name of the file.

Reply:
I do see your point here. This makes sense for table to have such a relationship and maintain table partitions, but does not for columns.

Reply:
fixed

Reply:
Do we even need this hive_column_partition ?

Reply:
That would be useful in the future but now is useless. Removed!